modules/doc/content/ncrc/applications/root_marmot.md

```diff
@@ -0,0 +1,19 @@
+## MARMOT
+
+!media application_logos/marmot.png style=width:8%;align:top;
+
+- [Documentation](https://marmot-docs.hpcondemand.inl.gov/latest/)
+- [Discussion Forum](https://marmot-discourse.hpcondemand.inl.gov)
+- [Build Status](https://civet.inl.gov/repo/851/)
+- [Level 1 - HPC OnDemand Execution](ncrc/applications/ncrc_marmot_ondemand.md)
+- [Level 1 - HPC Binary Execution](ncrc/applications/hpc_marmot.md)
+- [Level 2 - Local Binary Installation](ncrc/applications/conda_marmot.md)
+- [Level 4 - Source Access](ncrc/applications/ncrc_marmot_level4.md)
+
+!row!
+!col small=12 medium=8 large=8
+MARMOT is a mesoscale fuel performance code. As such, it can predict the evolution of the microstructure and material properties of fuels and claddings due to stress, temperature, and irradiation damage. MARMOT can, therefore, supply microstructure-based materials models to other applications that work at a larger scale, such as BISON which works at an engineering scale. MARMOT solves equations involving solid mechanics and heat conduction using the finite element method.
```

Comment:
```suggestion
MARMOT is a mesoscale fuel performance code. As such, it can predict the evolution of the microstructure and material properties of fuels and claddings due to stress, temperature, and irradiation damage. MARMOT can, therefore, supply microstructure-based materials models to other applications that work at a larger scale, such as BISON at the engineering scale. MARMOT solves equations involving solid mechanics and heat conduction using the finite element method.
```